### ISSUE TITLE
Unable to specify legend properties for items from secondary y-axis 

### ISSUE THREAD

**Issue by kchristman54** [EPPlusSoftware/EPPlus#522]
I am able to specify legend formatting for series on the primary y-axis. But not for series on the secondary y-axis. I don't know if this is a EPPlus bug, or I'm doing something wrong.

```c#
[Test]
        public void TestForDeletingLegendItemOnSecondaryYAxis()
        {
            //Adda a scatter chart on the data with one serie per row. 
            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
            using ExcelPackage package = new();
            ExcelWorksheet ws = package.Workbook.Worksheets.Add("Scatter Chart");
            
            CreateIceCreamData(ws);


            ExcelScatterChart chart = ws.Drawings.AddScatterChart("ScatterChart1", eScatterChartType.XYScatter);
            
            ExcelScatterChart secondAxis = (ExcelScatterChart) chart.PlotArea.ChartTypes.Add(chart.ChartType);
            secondAxis.UseSecondaryAxis = true;
            
            

            
            chart.SetPosition(1, 0, 3, 0);
            chart.To.Column = 18;
            chart.To.Row = 20;
            

            
            // Add two series to the primary y-axis
            ExcelScatterChartSerie serie11 = chart.Series.Add(ws.Cells[3, 2, 4, 2], ws.Cells[3, 1, 4, 1]);
            serie11.Header = "Primary #1";
            ExcelScatterChartSerie serie12 = chart.Series.Add(ws.Cells[3, 4, 4, 4], ws.Cells[3, 3, 4, 3]);
            serie12.Header = "Primary #2";
            
            // Add two series to the secondary y-axis
            ExcelScatterChartSerie serie21 = secondAxis.Series.Add(ws.Cells[3, 6, 4, 6], ws.Cells[3, 5, 4, 5]);
            serie21.Header = "Secondary #1";
            ExcelScatterChartSerie serie22 = secondAxis.Series.Add(ws.Cells[3, 8, 4, 8], ws.Cells[3, 7, 4, 7]);
            serie22.Header = "Secondary #2";


            // Setting the properties for legend items on the primary y-axis is straightforward:
            chart.Legend.Entries[0].Font.Size = 15;
            chart.Legend.Entries[1].Font.Size = 6;
            
            
            // But setting the properties for legend items from a secondary-yaxis, applies it to the primary items.
            secondAxis.Legend.Entries[0].Font.Italic = true;
            secondAxis.Legend.Entries[1].Font.Strike = eStrikeType.Single;

            // And if I try to access them as "3" and "4" on the main chart's Legend, results in an exception (if you un-comment the next two lines.
            //chart.Legend.Entries[3].Font.Italic = true;
            //chart.Legend.Entries[4].Font.Strike = eStrikeType.Single;
            
           
            
            File.WriteAllBytes(@"C:\dev\epplusExample.xlsx", package.GetAsByteArray()); 
        }
        
        protected static void CreateIceCreamData(ExcelWorksheet ws)
        {
            ws.SetValue("A3", new DateTime(2019, 1, 1));
            ws.SetValue("B3", 2500);
            ws.SetValue("A4", new DateTime(2019, 2, 1));
            ws.SetValue("B4", 3000);
            
            ws.SetValue("C3", new DateTime(2019, 3, 1));
            ws.SetValue("D3", 2700);
            ws.SetValue("C4", new DateTime(2019, 4, 1));
            ws.SetValue("D4", 4400);
            
            ws.SetValue("E3", new DateTime(2019, 5, 1));
            ws.SetValue("F3", 2700);
            ws.SetValue("E4", new DateTime(2019, 6, 1));
            ws.SetValue("F4", 4400);
            
            ws.SetValue("G3", new DateTime(2019, 7, 1));
            ws.SetValue("H3", 2700);
            ws.SetValue("G4", new DateTime(2019, 8, 1));
            ws.SetValue("H4", 4400);
            
        }

```

Running the above results in the italicizing and strike-through being applied to the series on the primary y-axis, not on the secondary. 
![image](https://user-images.githubusercontent.com/33758481/139483204-fd8685bf-f9cc-48fe-ab46-749276510e5d.png)

**Comment by JanKallman** [comment]
I'll have a look at this

**Comment by JanKallman** [comment]
Fixed in EPPlus 5.8.3
